src/components/UserDropdown.vue

```diff
@@ -87,6 +87,7 @@ export default defineComponent({
     border-radius: 3px;
```

Comment:
nit: what do you think about squaring the top corners here so it looks more like it is attached to the top bar?
```suggestion
    border-radius: 3px;
    border-top-left-radius: 0; 
    border-top-right-radius: 0;
```